server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -1329,6 +1330,15 @@ public DocsStats docStats() {
         return getEngine().docStats();
     }
 
+    public UnreferencedFileCleanUpStats unreferencedFileCleanupStats() {
```

Comment:
Is this method being called? If not, let's remove.

Reply:
Addressed the comment